### PR TITLE
docs: reflect removal of create and rename of init to create

### PR DESCRIPTION
**Pull request by lkingland**
Updates docs to reflect the replacing of the `create` command with `init`.

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -78,12 +78,14 @@ mkdir fn.example.io
...
 
-Now, with one command we will create the project files, build a container, and
+Now, we will create the project files, build a container, and
 deploy the function as a Knative service.
 
 
 ```bash
 faas create -l node
+faas build
+faas deploy
```
Comment (lkingland) [inline tutorial.md:88]: I am not sure if this is still correct. I expect to be testing this shortly, but perhaps a reviewer can chime in to expedite the process.

File: FAQ.md [hunk 2]
```diff
@@ -10,8 +10,7 @@ project are listed here.
 
 * Function runtimes for Go, Node.js and Quarkus
 * Buildpacks for Go, Node.js and Quarkus
-* A `faas` CLI for initializing, creating and deploying functions as a Knative
-  Service
+* A `faas` CLI for creating and managing functions as a Knative Service
```
Comment (lance) [inline FAQ.md]:
```suggestion
* A CLI for creating and managing functions as a Knative Service
```

File: tutorial.md (hunk 1 above)
Comment (lance) [reply tutorial.md:88]: This is correct, yes.

**Review by lance — COMMENTED** [review]
LGTM - just a couple of minor nits

**Pull request merged**